projects/1.20/assets/tacz-plus-timeless-and-classics-feature-addon/taczplus/lang/zh_cn.json

```
@@ -0,0 +1,20 @@
+{
+    "key.taczplus.deadeye": "死神之眼",
+    "subtitles.deadeyebeat": "死神之眼",
+    "key.taczplus.focus": "屏息",
+    "subtitles.deadeye": "死神之眼",
+    "key.taczplus.lean_left": "左探头",
+    "subtitles.exitdeadeye": "死神之眼：退出",
+    "effect.taczplus.lean_right_effect": "右探头",
+    "effect.taczplus.deadeye_effect": "死神之眼",
+    "gamerule.shakeDisable": "抖动",
+    "effect.taczplus.focus_effect": "屏息",
+    "key.taczplus.lean_right": "右探头",
+    "subtitles.deadeyeexit": "死神之眼：退出",
+    "gamerule.shakeDisable.description": "持枪时发生的抖动",
+    "gamerule.deadeyeDisable": "禁用左倾",
+    "effect.taczplus.leanlefteffect": "左探头",
+    "gamerule.deadeyeDisable.description": "禁用死神之眼",
+    "subtitles.deadeyebeatlouder": "死神之眼",
+    "item.taczplus.stim": "AI-2 医疗包"

```

```suggestion
    "item.taczplus.stim": "AI-2医疗包"
```
    在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。